### PR TITLE
javascript carousel

### PR DESCRIPTION
**Pull request by Adam-SungMin-Park**
Screen Shot 2020-12-10 at 13 10 16

### CODE REVIEW

File: javascript-carousel/main.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    $img5.className = "img5"
+    $img2.className = "hidden"
+    $img3.className = "hidden"
+    $img4.className = "hidden"
+    $img1.className = "hidden"
+    $circle5.className = "selected far fa-circle"
+    $circle2.className = "far fa-circle"
+    $circle3.className = "far fa-circle"
+    $circle4.className = "far fa-circle"
+    $circle1.className = "far fa-circle"
```
Comment (uzair-ashraf) [inline javascript-carousel/main.js:169]: This kind of repeating logic was the reason for loops were created, please refactor with a for loop.

File: javascript-carousel/main.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+var $circle4 = document.querySelector('#circle4');
+var $circle5 = document.querySelector('#circle5');
+
+
+var $rightButton = document.querySelector('#right');
+var $leftButton = document.querySelector('#left');
+
+var interval = null;
+
+interval = window.setInterval(imageChange,3000);
```
Comment (uzair-ashraf) [inline javascript-carousel/main.js:18]: image

When the user hasn't clicked for 2 seconds, and then clicks, then one second later the image will change again, you need something to reset the interval if the user clicks.

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

File: javascript-carousel/main.js (hunk 1 above)
Comment (Adam-SungMin-Park) [reply javascript-carousel/main.js]: updated

File: javascript-carousel/main.js (hunk 2 above)
Comment (Adam-SungMin-Park) [reply javascript-carousel/main.js:8]: updated

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**